sdk/nodejs/provider/experimental/analyzer.ts

```diff
@@ -55,6 +56,12 @@ interface docNode extends typescript.Node {
     jsDoc?: typescript.JSDoc[];
 }
 
+enum PulumiType {
```

Comment:
Not sure if there's a better name for this :thinking: I was struggling to come up with a name.

Reply:
Direction? InputOutput?

What does None mean? Unknown? Undefined? Both?

Reply:
Renamed it to `InputOutput`.  We can always find a better name still later if someone comes up with it.  Called `None`, `Neither` now, as in it's neither an input nor an output.